ww3tools/modelBuoy_collocation.py

```diff
@@ -88,747 +88,1003 @@
   and ww3) to maximize the amount of matchups even when one variable is 
   not available.
 
+.spec update: 
+
+How to run .spec file in a format of the .gz file: 
+Here is an example of how the job script should look like:
+# Define variables
+input_gz_file="multi_1.t11z.spec_tar.gz"
+output_directory="./" (any directory)
+buoy_path="/scratch2/NCEPDEV/marine/Matthew.Masarik/dat/buoys/NDBC/ncformat/wparam" (This path is on Hera. The path for Orion is different) 
+
+# Process data command
+python3 modelBuoy_collocation.py unzip "$input_gz_file" "$output_directory" "$buoy_path"
```

Comment:
let's move this part up to usage, and add the expected output as well if it's not similar with what was already in the script. I am also thinking if we should even take this spec file capability completely out of the current script. It is confusing that there are two ways to run the code.